jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.vue

```diff
@@ -23,7 +23,7 @@
         ></v-text-field>
       </template>
     </v-slider>
-    <v-checkbox v-model="linked" label="Linked"></v-checkbox>
+    <v-checkbox v-show="false" v-model="linked" label="Linked"></v-checkbox>
```

Comment:
```suggestion
    <!-- the below is temporarily hidden because of spacing issues, which will be adjusted in a future PR that makes more substantial changes to the UI layout -->
    <v-checkbox v-show="false" v-model="linked" label="Linked"></v-checkbox>
```